owncloudApp/src/main/java/com/owncloud/android/presentation/previews/PreviewTextViewModel.kt

```diff
@@ -38,11 +40,17 @@ class PreviewTextViewModel(
     private val filterFileMenuOptionsUseCase: FilterFileMenuOptionsUseCase,
     private val contextProvider: ContextProvider,
     private val coroutinesDispatcherProvider: CoroutinesDispatcherProvider,
+    getFileByIdAsStreamUseCase: GetFileByIdAsStreamUseCase,
+    ocFile: OCFile
```

Comment:
Let's use trailing commas to keep versioning cleaner. More info: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Done!